packages/yoastseo/spec/scoring/assessments/seo/ProductIdentifiersAssessmentSpec.js

```diff
@@ -152,3 +158,59 @@ describe( "a test for the applicability of the assessment", function() {
 	} );
 } );
 
+describe( "test the applicabilityHelper", () => {
```

Comment:
Maybe all unit tests inside this block can be moved to "a test for the applicability of the assessment" above?

Reply:
Good idea!